components/thumbnail/basic/src/index.js

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+
+class ThumbnailBasic extends Component {
+  render () {
+    const { image, captionText } = this.props
+    return (
+      <figure className='sui-ThumbnailBasic'>
+        <img className='sui-ThumbnailBasic-Image' src={image.src} alt={image.alt} />
```

Comment:
`sui-ThumbnailBasic-Image` should be `sui-ThumbnailBasic-image`.
same for `sui-ThumbnailBasic-Caption` => `sui-ThumbnailBasic-caption`

more info https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md